src/app/components/PopUpBox/PopUpBox.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Typography from '../Typography/Typography';
+import Button from '../Button/Button';
+import type { ReactNode } from 'react';
+import style from './PopUpBox.module.css';
+
+export type PopUpBoxProps = {
+  children: ReactNode;
+  boxType: 'infoBox' | 'statusBox';
+  className?: string;
+  onClose: () => void;
+  changeKeimen: () => void;
+  changeEinweichen: () => void;
+};
+
+const PopUpBox = ({
+  children,
+  className,
+  boxType,
+  onClose,
+  changeKeimen,
+  changeEinweichen,
```

Comment:
I think if these were in English, it would be better. Maybe `onGerminateClick` and `onSoakClick`?

Reply:
Done. Thank you